src/test/java/uk/gov/hmcts/reform/finrem/caseorchestration/handler/policy/ApplicantChangeOrgPolicyAboutToSubmitHandlerTest.java

```diff
@@ -97,4 +97,19 @@ private FinremCallbackRequest buildCallbackRequest() {
                 .data(finremCaseData).build())
             .build();
     }
-}
\ No newline at end of file
+
+    private static void assertCanHandle(CallbackHandler handler, Arguments... combination) {
```

Comment:
Just call the method in the Assertions class. You don't need to copy it to your class